notes.bash_completion

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+_notes_complete_notes() {
+    local configured_dir=${NOTES_DIRECTORY%/} # Remove trailing slashes
+    local notes_dir="${configured_dir:-$HOME/notes}"
+    local items=($(compgen -f "$notes_dir/$1"))
+    for item in "${items[@]}"; do
+        [[ -d $item ]] && item="$item/"
+        local filename=${item#$notes_dir/}
+        COMPREPLY+=("${filename%.md}")
+    done
```

Comment:
Hmm, I managed to fix it for spaces however I do not know how to deal with newlines in filenames still.

Should also not be that tricky to remove showing hidden files